### PR TITLE
Loadshed script

### PR DESCRIPTION
**Pull request by JulianaCP-v2**

### CODE REVIEW

File: operator_toolkit/loadshed.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+)
+
+CONFIG = load_config()
+
+
+def loadshed_site(site):
+    report = {}
+
+    # basic virtual operator url
+    virtual_operator_url = f'https://virtual-operator-{site}.axiomcloud.io'
```
Comment (lucasmlofaro) [inline operator_toolkit/loadshed.py:25]: Seems like a utility for constructing endpoint URLs would come in handy. This is probably done in a number of places and isn't the right place to define these URLs.

File: operator_toolkit/loadshed.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    gains = f'{rack_control_url}/gains/SoC_25_percentile'
+    resp = requests.put(gains, data=json.dumps(CONFIG["gains"]["SoC_25_percentile"]),
+                        auth=HTTPBasicAuth(AXIOM_API_AUTH_USER, AXIOM_API_AUTH_PASSWORD))
+    report["set_soc_25_percentile_gains"] = f"Status Code: {resp.status_code}"
+
+    # step 5: set SoC_50_percentile gains
+    gains = f'{rack_control_url}/gains/SoC_50_percentile'
+    resp = requests.put(gains, data=json.dumps(CONFIG["gains"]["SoC_50_percentile"]),
+                        auth=HTTPBasicAuth(AXIOM_API_AUTH_USER, AXIOM_API_AUTH_PASSWORD))
+    report["set_soc_50_percentile_gains"] = f"Status Code: {resp.status_code}"
```
Comment (lucasmlofaro) [inline operator_toolkit/loadshed.py:56]: There's a lot of redundant code here, this being a prime example. Don't necessarily have to do anything about it now.

File: operator_toolkit/loadshed.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                        help="Full path of the file that contains the loadshed rule.",
+                        type=str,
+                        required=True)
+
+    parser.add_argument('--controller',
+                        help=CLI_MESSAGE["controller"]["message"],
+                        type=str,
+                        choices=CLI_MESSAGE["controller"]["choices"])
+
+    args = parser.parse_args()
```
Comment (lucasmlofaro) [inline operator_toolkit/loadshed.py:108]: Is the intent right now to make a script into a CLI tool? Ideally I think we'd like to make a single CLI for all of the scripts, but this is a decent first step towards that.

Comment (JulianaCP-v2) [reply operator_toolkit/loadshed.py:108]: yes, the idea is to add a CLI on each python file (each python file is going to references a process that wants to be handle)

File: operator_toolkit/loadshed.py (hunk 2 above)
Comment (JulianaCP-v2) [reply operator_toolkit/loadshed.py:58]: 👌 After we have a basic improvement for each script. think we can work in a more fancy way to organize all the scripts and their requests.

**Review by lucasmlofaro — APPROVED** [review]

**Pull request merged**